setup.py

```diff
@@ -26,7 +26,7 @@
     author_email='data-analytics-team@palmetto.com',
     url='https://github.com/palmetto/palm-dbt',
     packages=find_namespace_packages(include=['palm', 'palm.*']),
-    package_data={'': ['*.md', '*.sql', '*.yaml', '*.yml']},
+    package_data={'': ['*.md', '*.sql', '*.yaml', '*.yml', '*.txt']},
```

Comment:
I was thinking about this a bit last night. I wonder if we could do something like this instead?

`package_data={'': ['palm/plugins/dbt/**/*']},`

I haven't tested that, but it seems _way_ cleaner to me